freqtrade/strategy/interface.py

```diff
@@ -276,12 +274,13 @@ def stop_loss_reached(self, current_rate: float, trade: Trade, current_time: dat
         """
 
         trailing_stop = self.config.get('trailing_stop', False)
-
         trade.adjust_stop_loss(trade.open_rate, force_stoploss if force_stoploss
                                else self.stoploss, initial=True)
 
-        # evaluate if the stoploss was hit
-        if self.stoploss is not None and trade.stop_loss >= current_rate:
+        # evaluate if the stoploss was hit if stoploss is not on exchange
+        if self.stoploss is not None and \
```

Comment:
consider wrapping everything in parenteses instead of using \ to wrap lines ...

Reply:
I have still difficulties with that shit, I don't know how to break lines ... but ok I will :)